ajax/libs/qcobjects/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "qcobjects",
+  "version": "2.1.298",
```

Comment:
Version should be omitted unless you wish to include the correct files for this version in the PR.

Reply:
Cool, I've added homepage, repository, bugs, author and keywords attributes